composer.json

```diff
@@ -54,8 +55,7 @@
             "drupal/console-extend-plugin": true,
```

Comment:
Can probably remove this line as well
```suggestion
```

https://packagist.org/packages/drupal/console-extend-plugin